sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -305,7 +303,8 @@ def test_big_query_new_types_native(self):
         'use_standard_sql': False,
         'native': True,
         'wait_until_finish_duration': WAIT_UNTIL_FINISH_DURATION_MS,
-        'on_success_matcher': all_of(*pipeline_verifiers)
+        'on_success_matcher': all_of(*pipeline_verifiers),
+        'experiments': 'use_dataflow_bq_sink',
```

Comment:
Ditto.

Reply:
I'm trying to preserve the same test coverage after the change, and remove BQSink tests later on. Would htis be fine?